src/components/Tooltip.vue

```diff
@@ -66,6 +87,11 @@ class Tooltip extends Vue {
     })
     public theme!: Tooltip.Themes;
 
+    /**
+     * Styles to apply on the tooltip box without the need to use deep css selectors.
+     */
+    @Prop(Object) public styles?: Partial<CSSStyleDeclaration>;
```

Comment:
Well, the styles are then not in the CSS file anymore, but in the HTML as inline-styles. It might be useful for small changes.

Reply:
The difference is that the prop is part of the component's public api, while deep css selectors work on the component's internals.